guide/README.md

```diff
@@ -0,0 +1,285 @@
+
+Calculate the pressure after several iterations using `gmx energy` and compare it with the IBI tutorial. Again, you can do this by copying the `Extract_Energies.sh` script from the main folder (`csg-tutorials/spce/ibi/`) to the appropriate step folder (`step_xxx`) and executing it. For a more detailed description of the RE method, look at the following [publication](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0131754).
+
+
+# Force matching for SPC/E water
+
+We will now derive a non-bonded CG potentials for SPC/E using the force matching method. 
+
+## Atomistic simulation
+
+Basis for the force matching procedure is an atomistic MD simulation. All files are found in the atomistic folder (`csg-tutorials/spce/atomistic/`). If you have done the above tutorials, you already generated the files of the atomistic md run. If not, then do so. Adjust the number of time steps to a reasonable value (`5000-10000`). and also the output frequency of the trajectory. Make sure, both, coordinates and forces are written to the trajectory file (`nstxout` and `nstfout` should have the same value in in `grompp.mdp`).
+
+## Force matching (FM)
+
+All files fore running the actual force matching calculation can be found in  `csg-tutorials/spce/force_matching/`. Have a look at the settings file (`fmatch.xml`). In the general force matching section (`cg.fmatch`), the number of frames to read in simultaneously (`frames_per_block`) and the type of LS solver (constrainedLS) are fixed (it is preferred to yous constrained LS). The fmatch block of the interaction (`cg.non-bonded.fmatch`) contains the interaction range (`min` and `max`), the step size for the internal spline representation (`step`) and the output step (`out_step`). `min` and `max` and have to be adjusted to be within the range of the RDF (see the calculating of RDF section of the IBI tutorial). Run the FM calculation (see also the corresponding line of the `run.sh` script): 
+
+```bash
+csg_fmatch --top ../atomistic/topol.tpr --trj ../atomistic/traj.trr --options fmatch.xml --cg water.xml (--begin $equi ) # ( ) denotes additional option: , --begin # time frame in GROMACS units (ps) to start analysis (e.g. 1 to 5)
+```
+
+To obtain the CG potential, the CG force has to be integrated. (see the appropriate lines in the `run.sh` script):
+
+```bash
+csg_call table integrate CG-CG.force CG-CG.pot # integrates the table
+csg_call table linearop CG-CG.pot CG-CG.pot -1 0 # multiplication of all table values by -1 (potential)
+```
+
+Change the spline grid (`step`), blocksize and parameter constrainedLS. This should get a feeling for the whole procedure.
+
+## Running of CG simulation
+
+To run a CG simulation with GROMACS, the potential has to be converted to a potential table, GROMACS can read (`table_CG_CG.xvg`). (Check the inverse section in the `fmatch.xml` for the corresponding options):
```

Comment:
"... to a potential table, GROMACS can read (`table_CG_CG.xvg`)." - > "... to a potential table, such as `table_CG_CG.xvg`.